flake.nix

```diff
@@ -370,35 +336,32 @@
     in
     rec {
       packages = {
-        default = packages.build;
-        build = buildNickel { };
-        buildWasm = buildNickelWasm { };
-        dockerImage = buildDocker packages.build; # TODO: docker image should be a passthru
+        nickel = (mkCraneArtifacts { }).nickel;
+        default = packages.nickel;
+        nickelWasm = buildNickelWasm { };
```

Comment:
I took the liberty to rename the Nix packages into what I think are more sensible names:
* `build` -> `nickel`
* `buildWasm` -> `nickelWasm`